packages/postcss/postcss.js

```diff
@@ -24,14 +24,18 @@ module.exports = postcss.plugin("modular-css", (opts) =>
                 return processor.output();
             })
             .then((output) => {
-                var { json } = processor.options;
-                
+                var { json, to } = processor.options;
+
                 result.messages.push({
                     type    : "modular-css-exports",
                     exports : classes,
                 });
                 
                 if(json) {
+                    if(typeof json !== "string") {
+                      json = to.replace(path.extname(to), ".json");
```

Comment:
This indentation doesn't look like 4 spaces

Reply:
Using `.replace` makes me a little nervous about files w/ weird names or paths, but is probably fine.

Reply:
Yeah, if it weren't for `path.extname` I wouldn't really trust it. If you'd prefer, can go to e.g. `${to}.json` instead.

Reply:
Because, for example, I should probably ensure it's the very last one, rather than the first one, that gets replaced.

Reply:
I see what you mean about its readability. But I agree that it's preferable to, for example:

```js
json = to.replace(new RegExp(`${path.extname(to)}$`), ".json");
```

It's actually clearer what it's doing and probably safer. Updated!